R/stats.R

```diff
@@ -27,6 +27,13 @@ n_complete <- function(x) {
   length(x) - n_missing(x)
 }
 
+#' @describeIn stats Calculate the sum of whitespace values.
```

Comment:
Can you provide a little more detail on what "whitespace" means. My first impression was that this would check for white spaces within words. I now know that this is wrong, but I wouldn't be surprised if others are mixed up this way.